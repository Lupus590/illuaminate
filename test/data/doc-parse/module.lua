--- @module foo

--- @module[library] foo

--- @module[module] foo

--- @module[unknown] foo
