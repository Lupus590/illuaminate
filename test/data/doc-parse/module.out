((source module.lua[1:1-1:15]) (module foo))
((source module.lua[3:1-3:24]) (module foo))
((source module.lua[5:1-5:23]) (module foo))
((source module.lua[7:1-7:24]) (module foo))
doc:unknown-flag: @module has unknown flag 'unknown'